src/controllers/places/get_specific.js

```diff
@@ -8,8 +8,15 @@ module.exports = (req, res) => {
     const place = JSON.parse(body);
     place.local = place[req.params.lang];
 
-    res.render('place', {
-      place
-    });
+    if (req.url.includes('edit')) {
```

Comment:
I think this `if` statement overcomplicates things. I would prefer to have two controllers:
```js
// get_specific.js
const Request = require('request');
const { placesURL } = require('../../constants/urls.json');

module.exports = (req, res) => {
  Request(`${placesURL}/${req.params.id}`, (error, response, body) => {
    if (error) return res.status(404).send(error);

    const place = JSON.parse(body);
    place.local = place[req.params.lang];
    res.render('place', {
        place
    });
  });
};
```

and 

```js
// edit_specific.js
const Request = require('request');
const { placesURL } = require('../../constants/urls.json');

module.exports = (req, res) => {
  Request(`${placesURL}/${req.params.id}`, (error, response, body) => {
    if (error) return res.status(404).send(error);

    const place = JSON.parse(body);
    place.local = place[req.params.lang];

    res.render('place-form', {
      place,
      edit: true
    });
  });
};
```